docs/container_types.rst

```diff
@@ -176,33 +176,62 @@ The dictionary terms are used as the chapter :ref:`slug` and the description of
 
 .. note:: Lengthy dictionary terms may be split into more than one chunk.
 
-The ``01.txt`` file contains the description of the term where the header is the title of the term and the rest is the description:
+The ``01.txt`` file contains the description of the term. The term title must always be at the top of the file as a
+h1 heading (a single #). :ref:`Links <linking>` may be used to reference other words, or content in other containers.
+
+.. note:: This type is in progress. We may add Strong's numbers to the markdown as well.
 
 .. code-block:: markdown
 
-    # Aaron
+    # Aaron #
+
+    ## Facts: ##
+
+    Aaron was Moses' older brother. God chose Aaron to be the first high priest for the people of Israel.
+
+    * Aaron helped Moses speak to Pharaoh about letting the Israelites go free.
+    * While the Israelites were traveling through the desert, Aaron sinned by making an idol for the people to worship.
+    * God also appointed Aaron and his descendants to be the [priests](kt/priest) for the people of Israel.
+
+    (Translation suggestions: [How to Translate Names](en/ta-vol1/translate/man/translate-names))
+
+    (See also: [[kt/priest]] **·** [[other/moses]] **·** [[other/israel]])
```

Comment:
These should be separated by commas.  I don't know who but the dots in there but the commas are much better.